crates/sqlexec/src/engine.rs

```diff
@@ -446,14 +446,26 @@ impl Engine {
             .storage
             .new_native_tables_storage(database_id, &storage)?;
         let state = metastore.get_cached_state().await?;
-        let catalog = SessionCatalog::new(
+        let mut catalog = SessionCatalog::new(
             state,
             ResolveConfig {
                 default_schema_oid: SCHEMA_DEFAULT.oid,
                 session_schema_oid: SCHEMA_CURRENT_SESSION.oid,
             },
         );
 
+        // If the database name is in the form of `<UUID>/<db_id>`, then we
+        // should use the <db_id> as the alias for the database.
+        let database_name = vars.database_name();
+        if let Some((org_id, db_id)) = database_name.split_once('/') {
+            if Uuid::parse_str(org_id).is_ok() {
+                catalog = catalog.with_alias(db_id.to_string());
+            }
+        // else, we should use the entire database name as the alias if it's not empty
```

Comment:
what happens in the the case that there's a problem parsing the org id? No alias? shouldn't we just ignore the org_id?

Reply:
are you referring to the edge case where the dbname contains a `/` but is not `<uuid>/db_id` from cloud?

Reply:
yes